### DISCUSSION TITLE
Make /dback icon in /homes have a blacklist world limit

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#9267] [Enhancement]
We are able to block the commands /dback and /back from an event, but we can not block pressing the button in the GUI.
We want to block peoples deaths in an event world, for them to be teleported home and using /homes and pressing the GUI button to get back to the event in the same death spot.

I cant do it in permissions, as they would be in a differnt world we want to block them to go to when they use it.

a setting could be set in homes.yml under WorldLimits for a new limit in which worlds dback and back can not be used in, a blacklist of worlds peolpe cant /back or /dback into from another world.

**Comment by Zrips** [comment]
Will make few changes for blacklists relating to back and dback commands with next update. Just keep in mind that clicking on Homes gui button is the same exact thing as performing /dback command directly as that button triggers command and all checks associated with it, which would include checking if destination location is in blacklisted world

**Reply by TomLewis** [reply]
Fantastic, where would I find the blacklists?

**Comment by TomLewis** [comment]
thanks for the update! now live!

you may have to update `cmi.command.back.worldbypass` as this no longer works.
